console-subscriber/src/aggregator.rs

```diff
@@ -543,6 +819,30 @@ impl<T> TaskData<T> {
     fn get(&self, id: &span::Id) -> Option<&T> {
         self.data.get(id).map(|(data, _)| data)
     }
+
+    fn as_proto(&mut self, include: Include) -> HashMap<u64, T::Result>
```

Comment:
nitpicky: generally, in Rust, [method names starting with `as_` are used only for zero-cost or low-cost conversions from a reference to a reference](https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv). this isn't a free reference conversion, so it should probably be called `to_proto` or `collect_proto` or something...